float64[] radius                  # Radius (further extend to full dim) [m]
int32 obstacle_count              # Total Obstacle Count